packages/main/src/components/AnalyticalTable/virtualization/VirtualTableBody.tsx

```diff
@@ -99,10 +102,27 @@ export const VirtualTableBody = (props) => {
     };
   }, [rows, visibleRows, minRows, props.rowHeight]);
 
+  const columnsWidth = useMemo(() => {
+    const aggregatedWidth = columns
+      .map((item) => {
+        if (
```

Comment:
I think this can be done easier by using `resizedColumns.hasOwnProperty(item.accessor)`